CMakeLists.txt

```diff
@@ -23,13 +23,13 @@ endif()
 
 # Use default toolchain file if not specified on the command line
 if(NOT CMAKE_TOOLCHAIN_FILE)
-  set(CMAKE_TOOLCHAIN_FILE ${CMAKE_SOURCE_DIR}/silkworm/cmake/toolchain.cmake CACHE FILEPATH "" FORCE)
+  set(CMAKE_TOOLCHAIN_FILE ${CMAKE_SOURCE_DIR}/silkworm/cmake/toolchain/cxx17.cmake CACHE FILEPATH "" FORCE)
```

Comment:
We could use `silkworm/cmake/toolchain/cxx20.cmake` for Silkrpc if this works for Silkworm too (it should)